apps/www/components/Article/PdfOverlay.js

```diff
@@ -4,103 +4,75 @@ import {
   Overlay,
   OverlayBody,
   OverlayToolbar,
-  A,
   Button,
   Checkbox,
   Radio,
 } from '@project-r/styleguide'
+
 import withT from '../../lib/withT'
-import { ASSETS_SERVER_BASE_URL } from '../../lib/constants'
-import { IconDownload } from '@republik/icons'
+
+import {
+  PUBLIC_BASE_URL,
+  SCREENSHOT_SERVER_BASE_URL,
+} from '../../lib/constants'
 
 export const getPdfUrl = (
-  meta,
-  { images = true, download = false, pageSize } = {},
+  { path, lastPublishedAt },
+  { images, pageFormat } = {},
 ) => {
-  const query = [
-    pageSize && pageSize !== 'A4' && `size=${pageSize}`,
-    !images && 'images=0',
-    download && 'download=1',
-  ].filter(Boolean)
-  return `${ASSETS_SERVER_BASE_URL}/pdf${meta.path}.pdf${
-    query.length ? `?${query.join('&')}` : ''
-  }`
-}
+  const pdfUrl = new URL(`${SCREENSHOT_SERVER_BASE_URL}/api/pdf`)
 
-const matchFigure = (node) =>
-  node.type === 'zone' && node.identifier === 'FIGURE'
-const matchVideo = (node) =>
-  node.type === 'zone' &&
-  node.identifier === 'EMBEDVIDEO' &&
-  node.data.forceAudio
+  const articleUrl = `${PUBLIC_BASE_URL}/${path}`
 
-export const countImages = (element) => {
-  if (matchFigure(element) || matchVideo(element)) {
-    return 1
-  }
-  return (element.children || []).reduce(
-    (count, node) => count + countImages(node),
-    0,
-  )
+  pdfUrl.searchParams.set('url', articleUrl)
```

Comment:
I think this is needed:
```suggestion
  pdfUrl.searchParams.set('url', encodeURIComponent(articleUrl))
```